app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -2,19 +2,27 @@
   # We have to call deep_symbolize_keys because we're often dealing with a
   # parsed JSON document which will have string keys by default, but our
   # components use symbol keys and we want consistency.
-  links_hash = content_item.to_h.deep_symbolize_keys[:links] || {}
+  content_item_hash = content_item.to_h.deep_symbolize_keys
+  links_hash = content_item_hash[:links] || {}
+  meta_tags = {}
+
+  meta_tags["govuk:format"] = content_item_hash[:format] if content_item_hash[:format]
+
+  need_ids = content_item_hash[:need_ids] || []
+  meta_tags["govuk:need-ids"] = need_ids.join(',') if need_ids.any?
 
   organisations = []
   organisations += links_hash[:organisations] || []
   organisations += links_hash[:lead_organisations] || []
   organisations += links_hash[:supporting_organisations] || []
   organisations += links_hash[:worldwide_organisations] || []
+  organisations_content = organisations.map{ |link| "<#{link[:analytics_identifier]}>" }.join
```

Comment:
We should have a space before the `{` as per the [styleguide](https://github.com/alphagov/styleguides/blob/master/ruby.md#general), same below.